### DISCUSSION TITLE
k8sgpt operator not reaching the LocalAI service

### DISCUSSION THREAD

**Discussion by wmashal** [mudler/LocalAI#470] [Q&A]
Hi 

The k8sgpt operator not reaching the LocalAI , I am using v0.3.4 and the enableAI is true but still see the following  errors in the k8gpt deployment and operator unless the LocalAI is up and running

![image](https://github.com/k8sgpt-ai/k8sgpt-operator/assets/11045408/7ff4ad23-a255-4d2c-a278-03c5c1946c12)

![image](https://github.com/go-skynet/LocalAI/assets/11045408/c68f4fc2-cfb6-451b-a8a1-1f3f1753abcb)

![image](https://github.com/k8sgpt-ai/k8sgpt-operator/assets/11045408/69d9b393-eba6-45f5-9077-12d860ad4fa3)

![image](https://github.com/k8sgpt-ai/k8sgpt-operator/assets/11045408/5c9e8cd6-e684-48ba-80c2-273504405956)

As you see I have the baseURL but also the operator still complaining

![image](https://github.com/k8sgpt-ai/k8sgpt-operator/assets/11045408/7eee1523-d1c0-4553-a33c-2983290f86d5)
 
Please Advice

**Comment by Aisuko** [comment]
Please check it here, https://github.com/go-skynet/LocalAI/issues/471#issuecomment-1573067582
